domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs

```diff
@@ -58,21 +86,28 @@ public async Task SeedData(Action<ApplicationDbContext> seeder)
         using var scope = Services.CreateScope();
         var dbContext = scope.ServiceProvider.GetRequiredService<ApplicationDbContext>();
 
-        dbContext.TransferAgreements.RemoveRange(dbContext.TransferAgreements);
-        await dbContext.SaveChangesAsync();
+        // Clear TransferAgreementAudits table
+        await dbContext.Database.ExecuteSqlRawAsync("TRUNCATE TABLE \"TransferAgreementAudits\"");
```

Comment:
The table name is a hardcoded value. Is it possible to avoid this, using e.g. `nameof`?